### PR TITLE
Try exponential representation of rate to make sqrt-balance possible

### PR DESCRIPTION
**Pull request by mschauer**

### CODE REVIEW

File: src/sampler.jl [hunk 1]
```diff
@@ -1,7 +1,8 @@
+using LogarithmicNumbers
 # Valid balancing functions
 
 metropolis_balance(t) = min(one(t), t)
-sqrt_balance(t) = sqrt(t)
+sqrt_balance(t) = t^(0.5)
```
Comment (mschauer) [inline src/sampler.jl:5]:
```suggestion
sqrt_balance(t) = sqrt(t)
```

Fixed by https://github.com/cjdoris/LogarithmicNumbers.jl/pull/19#issuecomment-1925811277

File: src/sampler.jl [hunk 2]
```diff
@@ -232,35 +243,43 @@ function causalzigzag(n, G = (DiGraph(n), 0); balance = metropolis_balance, prio
...
+        #λbar = max(dir*float(-s1 + s2), 0.0)
+        #λrw = float(s1 + s2) 
+        #λup = float(s1)   
+        λ = dir == 1 ? (s1) : (s2)
         λbar = max(dir*(-s1 + s2), 0.0)
         λrw = (s1 + s2) 
-        λup = s1   
-        λ = dir == 1 ? abs(s1) : abs(s2)
+        λup = (s1)   
+        λ = dir == 1 ? (s1) : (s2)
```
Comment (mwien) [inline src/sampler.jl:253]: λ is assigned twice the same value

File: src/sampler.jl (hunk 1 above)
Comment (mschauer) [reply src/sampler.jl]: If we leave sqrt then we can merge but we shouldn’t th a new version until upstream is released

Comment (mwien) [reply src/sampler.jl]: ok

**Pull request merged**